_limitations/set_limitations.md

```diff
@@ -0,0 +1,48 @@
+---
+layout: default
+title: Unsupported SET Statements 
+nav_order: 2
+---
+
+## Unsupported SET statements
+
+The following SET statements are available for use with Babelfish versions 1.2.0 and higher:
+
+- `SET FMTONLY` - `SET FMTONLY ON/OFF` applies only to `SELECT *` queries, and is ignored for all other queries.
+- `SET LOCK_TIMEOUT`
+
+The following SET statement limitations apply:
+
+- `SET LANGUAGE` - This syntax is not supported with any value other than `english` or `us_english`. 
+
+
+The following SET statements are not implemented by Babelfish: 
+
+- `SET ANSI_NULL_DFLT_OFF ON`
+- `SET ANSI_NULL_DFLT_ON OFF`
+- `SET ANSI_PADDING OFF`
+- `SET ANSI_WARNINGS OFF`
+- `SET ARITHABORT OFF`
+- `SET ARITHIGNORE ON`
+- `SET CONTEXT_INFO`
+- `SET CURSOR_CLOSE_ON_COMMIT ON`
+- `SET DATEFORMAT`
+- `SET DEADLOCK_PRIORITY`
+- `SET FORCEPLAN`
+- `SET NO BROWSETABLE`
+- `SET NUMERIC_ROUNDABORT ON`
+- `SET REMOTE_PROC_TRANSACTIONS`
+- `SET ROWCOUNT n WHERE n != 0`
+- `SET ROWCOUNT @variable`
+- `SET SHOWPLAN_ALL`
```

Comment:
this is supported in 2.2.0 under a new escape hatch

Reply:
This page hasn't been updated yet for 2.2.0 - I'd like to push the website formatting changes out with 2.1.0a, and then update the docs for Manatee.